client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
```

Comment:
Typically we prefer shallow to avoid the dom setup, but it looks like this component is complex enough to use mount.